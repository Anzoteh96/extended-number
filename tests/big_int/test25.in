-5207999181013867694400973571310588486551480959626297907827900785673674153052
-993781332178043531894362741253623871085620436371198294165467823591162441552
