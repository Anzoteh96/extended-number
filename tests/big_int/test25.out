Enter two numbers n and m:
n is -5207999181013867694400973571310588486551480959626297907827900785673674153052
m is -993781332178043531894362741253623871085620436371198294165467823591162441552
n + m is -6201780513191911226295336312564212357637101395997496201993368609264836594604
n - m is -4214217848835824162506610830056964615465860523255099613662432962082511711500
n * m is 5175612364090121116104226046050002659418095374604599750929296417676492138539975218421415222284146958212331665433430067360661025290551521244696052416704
n / m is 5
n % m is -239092520123650034929159865042469131123378777770306437000561667717861945292
